source/renderer/app/components/staking/redeem-itn-rewards/Step1ConfigurationDialog.js

```diff
@@ -0,0 +1,391 @@
+
+  submit = () => {
+    this.form.submit({
+      onSuccess: form => {
+        const { onContinue } = this.props;
+        const { recoveryPhrase } = form.values();
+        onContinue({
+          recoveryPhrase,
+        });
+      },
+      onError: () =>
+        handleFormErrors('.ConfigurationDialog_error', { focusElement: true }),
+    });
+  };
+
+  get walletsDropdownError() {
+    const { intl } = this.context;
+    const { wallet } = this.props;
+    let walletsDropdownError;
+    if (wallet && wallet.amount.isZero())
+      walletsDropdownError = intl.formatMessage(messages.walletsDropdownError);
+    return walletsDropdownError;
+  }
+
+  get canSubmit() {
+    const { isSubmitting, wallet } = this.props;
+    const { form } = this;
+    const { checked: checkboxAcceptance1isChecked } = form.$(
+      'checkboxAcceptance1'
+    );
+    const { checked: checkboxAcceptance2isChecked } = form.$(
+      'checkboxAcceptance2'
+    );
+    return (
+      !isSubmitting &&
+      wallet &&
+      !wallet.amount.isZero() &&
+      checkboxAcceptance1isChecked &&
+      checkboxAcceptance2isChecked &&
+      !this.walletsDropdownError &&
+      form.isValid
+    );
+  }
+
+  render() {
+    const { intl } = this.context;
+    const { form } = this;
+    const {
+      isSubmitting,
+      onClose,
+      onContinue,
+      onSelectWallet,
+      wallet,
+      suggestedMnemonics,
+      openExternalLink,
+      wallets,
+      error,
+    } = this.props;
+    const recoveryPhraseField = form.$('recoveryPhrase');
+    const walletsDropdownField = form.$('walletsDropdown');
+    const checkboxAcceptance1Field = form.$('checkboxAcceptance1');
+    const checkboxAcceptance2Field = form.$('checkboxAcceptance2');
+    const walletId = get(wallet, 'id', null);
+
+    const buttonClasses = classnames([
+      'primary',
+      isSubmitting ? styles.isSubmitting : null,
+    ]);
+
+    const actions = {
+      direction: 'column',
+      items: [
+        {
+          className: buttonClasses,
+          disabled: !this.canSubmit,
+          primary: true,
+          label: intl.formatMessage(messages.continueButtonLabel),
+          onClick: this.submit,
+        },
+        {
+          onClick: (event: MouseEvent) =>
+            openExternalLink(
+              intl.formatMessage(messages.learnMoreLinkUrl, event)
+            ),
+          label: intl.formatMessage(messages.learnMoreLinkLabel),
+          isLink: true,
+        },
+      ],
+    };
+
+    const itnLink = (
+      <Link
+        className={styles.itnLink}
+        onClick={(event: MouseEvent) =>
+          openExternalLink(
+            intl.formatMessage(messages.descriptionItnLinkUrl, event)
+          )
+        }
+        label={intl.formatMessage(messages.descriptionItnLinkLabel)}
+        skin={LinkSkin}
+      />
+    );
+
+    return (
+      <>
+        <DialogCloseButton
+          className={redeemDialogOverride.closeButton}
+          onClose={onClose}
+        />
```

Comment:
@daniloprates same as above. It would be better if we can add close dialog button as Dialog property.

Reply:
The issue was that the Dialog has `position: relative`, which prevented the button to be outside its box. But I had already an Override file, so it was easy to fix this. Done. Thanks.

Reply:
@daniloprates nice 👏